### ISSUE TITLE
mode collapse

### ISSUE THREAD

**Issue by hugo-xie** [NVlabs/imaginaire#43]
hi, when I trained with coco_funit, In the first few epochs, the results are normal, but mode collapse appears from the 59th epoch. Is this normal? Will it also appear during your training?
![epoch_00063_iteration_000094000](https://user-images.githubusercontent.com/15179916/101979545-d6b7d700-3c98-11eb-99f1-0b1eff6c50e1.jpg)

**Comment by hugo-xie** [comment]
any suggestion？

**Comment by Edwardkim0** [comment]
i'm also curious about that...

**Comment by mingyuliutw** [comment]
Mode collapse is still an open problem in GAN training, especially dealing with challenging distribution.

**Comment by Edwardkim0** [comment]
![image](https://user-images.githubusercontent.com/47470342/110403600-bb563f80-80c0-11eb-952c-cb988c1d01b3.png)
i have 2 gpus, and costom dataset.
but same problem... i will try batch_size 2 and then comment

**Comment by mingyuliutw** [comment]
Maybe lower the learning rate or increase the reconstruction loss?

**Comment by Edwardkim0** [comment]
any suggestion?

**Comment by Edwardkim0** [comment]
> any suggestion?

I collect more data about 10k each category. then mode collapse not happend.
